src/names/applications.cpp

```diff
@@ -183,4 +185,34 @@ IsValidJSONOrEmptyString (const std::string& text){
     UniValue v;
 
     return text.empty() || v.read(text);
+}    
+
+bool
+IsMinimalJSONOrEmptyString (const std::string& text){
+    UniValue v;
+    if(text.empty()){
+        return true;
+    } 
+
+    if(!v.read(text)){ 
+        return false;
+    } else {
```

Comment:
Per Daniel's comment about the above `else` usage, I think you can also avoid `else` here since the `if` block includes a `return`.